### ISSUE TITLE
About the input method box position

### ISSUE THREAD

**Issue by leavor** [spyder-ide/spyder#9957]
## Problem Description
In spyder 4.0.0b4,when inputting Chinese, the input box blocks the cursor,like this:
![image](https://user-images.githubusercontent.com/1759365/62443615-e9112f00-b78d-11e9-92ce-a1bc181fa827.png)

But in spyder3.3.6,it is ok,like this:
![image](https://user-images.githubusercontent.com/1759365/62443562-d0a11480-b78d-11e9-9534-165cdfe8801e.png)

Only the spyder version is different, all other environments are the same

## Versions

* Spyder version: 4.0.0b4
* Python version: 
* Qt version: 
* PyQt version: 
* Operating System name/version: win7

### Dependencies

```
PASTE DEPENDENCIES HERE
```

**Comment by ccordoba12** [comment]
I'm sorry but I don't know how we can fix this because I have no idea how that input box is shown. Is that part of an special software?

**Comment by leavor** [comment]
It is a Chinese input method software, named Sogou,popular in China!
In **spyder IDE 3.3.6**,everything is ok.

**Comment by leavor** [comment]
> I'm sorry but I don't know how we can fix this because I have no idea how that input box is shown. Is that part of an special software?

It is a Chinese input method software, named Sogou,popular in China!
In spyder IDE 3.3.6,everything is ok.

**Comment by ccordoba12** [comment]
Where can we download it?

**Comment by leavor** [comment]
> Where can we download it?

Can be downloaded here:[https://pinyin.sogou.com/](url)
![image](https://user-images.githubusercontent.com/1759365/62453783-9f344300-b7a5-11e9-8cfb-2fc8e7dd67a9.png)

The “Sogo” software attribute:
![image](https://user-images.githubusercontent.com/1759365/62454356-c17a9080-b7a6-11e9-91c8-35bcdb5cbd5e.png)

**Comment by ausir2016** [comment]
This issue exists in new version like Spyder IDE 5.4.3.

**Comment by ausir2016** [comment]
This issue exists in new version like Spyder IDE 5.4.3. 
* Spyder version: 5.4.3  (conda)
* Python version: 3.11.5 64-bit
* Qt version: 5.15.2
* PyQt5 version: 5.15.10
* Operating System: Windows 10

**Comment by dalthviz** [comment]
Stumbled with this issue and thinking about it maybe it could be related with https://github.com/spyder-ide/spyder/issues/24561 ? The fix for that one got released with Spyder 6.0.8 so maybe, even when the input software used here is different, updating/using Spyder 6.0.8 could help with this too? 🤔

**Comment by ccordoba12** [comment]
Yep, I think so too. Let's close this one because it's too old and assume it's fixed in our 6.0.8 version.
